dmoj/cptbox/sandbox.py

```diff
@@ -156,7 +163,7 @@ def __init__(self, debugger, _, args, executable=None, security=None, time=0, me
         if security is None:
             self._trace_syscalls = False
         else:
-            for i in xrange(SYSCALL_COUNT):
+            for i in range(SYSCALL_COUNT):
```

Comment:
Import six.moves.range and be done with it.

Reply:
Done